readme.html

```diff
@@ -531,7 +531,7 @@ <h1><a href="https://yourls.org/">YOURLS: <span>Y</span>our <span>O</span>wn <sp
 
 			<a class="anch" href="#FAQ;requirement"><span>&#10148;</span><h3 id="requirement">Server requirements and recommendations</h3></a>
 			<ol>
-				<li>We recommend <strong>PHP 7.4</strong> or above</li>
+				<li>We recommend <strong>PHP 8.1</strong> or above</li>
```

Comment:
Is it really still a recommendation, or is it a requirement now?

Reply:
I guess it's required now 😅
```suggestion
				<li><strong>PHP 8.1</strong> or above is required</li>
```